fs/operations/operations.go

```diff
@@ -502,7 +502,7 @@ func Copy(ctx context.Context, f fs.Fs, dst fs.Object, remote string, src fs.Obj
 		}
 	}
 
-	fs.Infof(src, actionTaken)
+	fs.Infof(src, fmt.Sprintf("%s %%s", actionTaken), newDst.String())
```

Comment:
You can write this

    fs.Infof(src, "%s to: %s", actionTaken, newDst)

I prefer the leading `to: `